### PR TITLE
Bug: Fix install risczero toolchain bug for x86-64-macos

### PR DESCRIPTION
**Pull request by clint419**
# Bug description
risc0/risc0#1507

## Fixed Issue
risc0/risc0#1512

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,11 @@ install-dev-tools:  ## Installs all necessary cargo helpers
 	cargo install cargo-nextest --locked
 	cargo install cargo-binstall
 	cargo binstall cargo-risczero
+ifeq ($(shell uname -mo), Darwin x86_64)
```
Comment (eyusufatik) [inline Makefile:32]: indent right please

**Review by eyusufatik — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (otaliptus) [inline Makefile:32]: Thanks for the PR!

I believe -o is not directly POSIX for `uname`. Just tested on an x86_64 machine, -s seems to be the correct one. Could you check if -s works, and then convert -o to -s?

So it should be `uname -ms`

**Review by otaliptus — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (clint419) [reply Makefile:32]: > indent right please

https://stackoverflow.com/questions/4483313/make-error-for-ifeq-syntax-error-near-unexpected-token

Comment (clint419) [reply Makefile:32]: > Thanks for the PR!
> 
> I believe -o is not directly POSIX for `uname`. Just tested on a Mac x86_64 machine, -o does not work | -s seems to be the correct one. Could you check if -s works, and then convert -o to -s?
> 
> So it should be `uname -ms`
> 
> Edit: [pystardust/ani-cli#416 (comment)](https://github.com/pystardust/ani-cli/issues/416#issuecomment-1023607443)

Yes, thank you a lot

Comment (otaliptus) [reply Makefile]: Great

**Pull request merged**